app/components/BuyRamForm/index.js

```diff
@@ -70,35 +101,57 @@ const FormObject = props => {
             }}
           />
         </GridItem>
-        <GridItem xs={12} sm={12} md={6}>
-          <CustomInput
-            labelText="Ram purchase (in EOS)"
-            id="quantity"
-            error={errors.quantity}
-            touched={touched.quantity}
-            formControlProps={{
-              fullWidth: true,
-            }}
-            inputProps={{
-              type: 'text',
-              placeholder: 'Required to process transactions',
-              value: values.quantity,
-              onChange: handleChange,
-              onBlur: handleBlur,
-            }}
+        <GridItem className={classes.radioContainer} xs={12} sm={12} md={6}>
+          <span className={classes.radioLabel}>Purchase unit:</span>
+          <FormControlLabel
+            value="eos"
+            control={<Radio checked={isEOS} color="primary" onChange={handleEOSUnitChange} />}
+            label="EOS"
+          />
+          <FormControlLabel
+            value="bytes"
+            control={<Radio checked={!isEOS} color="primary" onChange={handleByteUnitChange} />}
+            label="bytes"
           />
         </GridItem>
-        <GridItem xs={12} sm={12} md={4}>
-          <Button onClick={handleSubmit} color="rose">
-            Purchase
-          </Button>
+        <GridItem xs={12} sm={12} md={6}>
```

Comment:
I am keeping 2 different inputs, I believe this is the best way to handle this logic with `Formik`. But happy to hear any feedback